community/commercial-support.tt

```diff
@@ -187,6 +187,19 @@ development lifecycle.
         Turning operational issues into development tasks using Nix since 2013.
       </a>
     </li>
+
+    <li>
+      <a href="https://www.cyberus-technology.de/services?utm_source=nixos&utm_medium=referal&utm_campaign=nixos"/>
```

Comment:
```suggestion
      <a href="https://www.cyberus-technology.de/services"/>
```

Reply:
Two fixes:
- html fix (a element is closed later on)
- lets include pure links without any tracking help

Reply:
Sorry to nag, but the SVG needs to be smaller right now the logo canvas is too big and is messing up the page.

Reply:
Sorry for taking so long. I've updated the image. Can you please check again?